lib/setup.js

```diff
@@ -17,8 +17,10 @@ function setupTestServer (opt, cb) {
     setup()
   }
 
-  function setup (_supportServer) {
-    supportServer = _supportServer
+  function setup (err, server) {
+    if (err) return cb(err)
+
+    supportServer = server
```

Comment:
I'm wondering about this `supportServer` stuff and if we can get rid of it.

Reply:
It would mean we can throw away `user-server.js` completely.

Reply:
Also had that thought. I guess there are use cases for it, but the current implementation doesn't seem very solid. We could always bring it back if people need it (and then rethink the approach).

Reply:
I don't even know what it does. Is it for integrating with external systems?

Reply:
See https://github.com/airtap/airtap/blob/master/doc/airtap.yml.md#server-optional. You can use this feature to for example run a websocket server and run integration tests with airtap. Airtap will setup and teardown the server.

Reply:
The alternative is `start-server.js && airtap test.js && kill-server.js`.